lib/jazz_fingers.rb

```diff
@@ -1,9 +1,24 @@
 require 'jazz_fingers/version'
-require 'jazz_fingers/railtie' if defined?(Rails)
-require 'active_support'
 require 'readline'
+require 'awesome_print'
+require 'pry'
+require 'pry-coolline'
+require 'pry-doc'
+require 'pry-git'
+require 'pry-remote'
+require 'jazz_fingers/hirb_ext'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.